src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
 // write your code here
+const h1 = document.querySelector('.logo');
+const body = document.body;
+
+const promise2 = () =>
+  new Promise((resolve, reject) => {
+    setTimeout(() => {
+      const div = document.createElement('div');
+
+      div.classList.add('message', 'error-message');
+      div.textContent = `Promise was rejected!`;
+      body.append(div);
+    }, 3000);
+  });
+
+const promise1 = () =>
+  new Promise((resolve) => {
+    const div = document.createElement('div');
+
+    div.classList.add('message');
+    div.textContent = 'Promise was resolved';
+    body.append(div);
+    resolve();
+  }).then(promise2);
```

Comment:
You need to add success and error handlers for both promises. Currently, there is no error handler for `promise2`. You should chain a `.catch()` method to handle the rejection of `promise2`.